src/v/cluster/topics_frontend.cc

```diff
@@ -163,6 +163,10 @@ ss::future<std::vector<topic_result>> topics_frontend::update_topic_properties(
         co_return create_topic_results(updates, errc::no_leader_controller);
     }
 
+    if (!_features.local().is_active(features::feature::cloud_retention)) {
```

Comment:
is it intended to reject all updates when `cloud_retention` feature is not active ?